webchat/auth/authService.js

```diff
@@ -152,6 +152,7 @@
             delete $window.localStorage.userInfo;
             userInfo = undefined;
             clearInterval(refreshInterval);
+            chatClient.closeClient();
```

Comment:
this should be called when the user closes the window too?

Reply:
No, the websocket does this automatically.